.github/workflows/nightly-macos-test.yml

```diff
@@ -47,8 +47,46 @@ jobs:
           test-name: macos-13-${{ matrix.build_type }}-tests
           cachix-auth-token: ${{ secrets.CACHIX_AUTH_TOKEN }}
 
+  motoko-core-tests:
+    needs: systems-go-tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get `motoko` Repo
+        uses: actions/checkout@v4
+      - name: Populate `moc`
+        uses: ./.github/actions/test-blueprint
+        with:
+          os: ubuntu-latest
+          test-target: moc
+          test-name: fetch-moc
+          cachix-auth-token: ${{ secrets.CACHIX_AUTH_TOKEN }}
+      - name: Build `moc`
+        run: |
+          nix build .#moc --print-out-paths
+      - name: Save the `test-blueprint` action
+        run: cp -r ./.github/actions/test-blueprint ..
```

Comment:
not super clear to me: why do we need to copy the blueprint somewhere else?

Reply:
Because the checkout of `motoko-core` zeroes out the previous checkout.